.github/workflows/tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         with:
           v-version: latest
         id: v
-      - name: Build repository app
-        run: v . -shared
+      # - name: Build repository app
+      #   run: v . -shared
```

Comment:
is there any point to let this two comment line in this file ?

Reply:
indeed